pycbc/transforms.py

```diff
@@ -612,7 +612,101 @@ def inverse_jacobian(self, maps):
         ref_mass=1.4
         mchirp = maps['mchirp']
         return (2.**(-1./5) * self.ref_mass / mchirp)**(5./6)
+    
+######################################################################################################### TEST ################################################################################################################################################
+import numpy as np
+from pycbc.waveform.waveform_modes import jframe_to_l0frame
+
+class alignTotalSpin(BaseTransform):
+    
+    """Parameters
+    ----------
+    iota =
+    
+    s1 =
+    
+    s2 =
+    
+    m1 =
+    
+    m2 =
+    
+    f_lower =
+    
+    phi_ref =
+    """
+    name = "Align_total_spin"
+    
+    def __init__(self, iota, s1, s2, m1, m2, f_lower, phi_ref, newIota, news1, news2):
+        self.iota = iota
+        self.s1 = s1
+        self.s2 = s2
+        self.m1 = m1
+        self.m2 = m2
+        self.f_lower = f_lower
+        self.phi_ref = phi_ref
+        self.newIota = newIota
+        self.news1 = news1
+        self.news2 = news2
+        self._inputs = [self.iota, self.s1, self.s2, self.m1, self.m2, self.f_lower, self.phi_ref]
+        self._outputs = [self.newIota, self.news1, self.news2]
+        super(alignTotalSpin, self).__init__()   ############################################## what??????
```

Comment:
Why the comment here? (While we are still using python2 we do need to support old-style super statements .... python2 will be retired soon though, so feel free to update once we do that!)